[Text Fragments article] Add opt-out steps (#3553)
GoogleChrome/web.dev: src/site/content/en/blog/text-fragments/index.md

Document-Policy: force-load-at-top
```

## Disabling text fragments

> In case you really dislike the feature, you can opt out by leveraging the enterprise setting

Reviewer: ```suggestion
If you really dislike the feature, you can opt out using the enterprise setting
```

[`ScrollToTextFragmentEnabled`](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ScrollToTextFragmentEnabled).

> Therefore, on macOS, paste the command below in the terminal.

Reviewer: ```suggestion
To use it on macOS, paste the command below in the terminal.
```

```bash
defaults write com.google.Chrome ScrollToTextFragmentEnabled -bool false
```

On Windows, follow the documentation on the 

> [Google Chrome Enterprise Help](https://support.google.com/chrome/a/answer/9131254?hl=en).

Reviewer: ```suggestion
[Google Chrome Enterprise Help](https://support.google.com/chrome/a/answer/9131254?hl=en) support site.
```
